dandi/files.py

```diff
@@ -518,21 +518,27 @@ def get_validation_errors(
         else:
             # make sure that we have some basic metadata fields we require
             try:
-                meta = get_metadata(self.filepath)
+                errors.extend(
+                    [
+                        error.message
+                        for error in inspect_nwb(
+                            nwbfile_path=self.filepath,
+                            skip_validate=True,
+                            config=load_config(filepath_or_keyword="dandi"),
+                            importance_threshold=Importance.CRITICAL,
```

Comment:
Now, looking towards the future you envision for easily extending the behavior of `dandi validate` to include the list of extra metadata checks that you want (Icephys cell things, ontologies, specific forms of subject fields, etc): these would be very easy to include simply by following the comments I left in the config file. That is, when you are all ready for that change, you can 
Now, we do _strongly_ recommend however that, for further customization and control for any `dandi validate`-exclusive purposes, you should move that config file (or make your own) over to live on the DANDI side of things. I believe this is the best option for both of us, since it is more general than you having to hard-code a list of explicit checks or metadata fields (you automatically inherit any additional checks/debugs we ever add to the NWBInspector) but it would also soften _your own_ dependency on _our_ release cycle if you want to make any on-the-fly adjustments to  metadata considerations.

Either way, this integration is perhaps beginning to err on the side of meriting a joint meeting - please schedule with Ben if you'd like to discuss all this in person.